test/cqlpy/test_collection_subscript.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+
+#############################################################################
+# Tests for SELECT of a specific key in a collection column
```

Comment:
Nitpick: There is a slight disagreement between this comment (saying it's only on SELECT) and the test file's name (suggesting maybe tests for IF or WHERE with subscripts also belong in this file). If you make this file only about select (like you said in the comment) you can call it something like test_select_subscript.py or something.

Reply:
I renamed the test file to `test_select_collection_element.py`